sdk/search/azure-search/src/main/java/com/azure/search/SearchApiKeyCredential.java

```diff
@@ -30,4 +30,15 @@ public SearchApiKeyCredential(String apiKey) {
     public String getApiKey() {
         return apiKey;
     }
+
+    /**
+     * Rotates the credential on the same Search Client.
```

Comment:
Credentials may be shared across clients, so this will affect all clients using this credential.

```suggestion
     * Rotates the Search credential.
```